charts/misc-apps/values.yaml

```diff
@@ -33,7 +33,7 @@ signalilo:
   # -- Chart
   chart: "signalilo"
   # -- [signalilo Helm chart](https://github.com/appuio/charts/tree/master/appuio/signalilo) version
-  targetRevision: "0.8.*"
+  targetRevision: "0.9.*"
```

Comment:
as mentioned in a mail from last week, please remove `wildcard` from targeted charts.